<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigTerminalWindowWidget</class>
 <widget class="QWidget" name="ConfigTerminalWindowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>New line</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Receive</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Transmit</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>小arriage return</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Line Feed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>小arriage return + Line feed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Automatic</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>小arriage return</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Line feed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>小arriage return + Line feed</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Size</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="lblSizeFrame">
        <property name="text">
         <string>Frame</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,1">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>W:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox">
          <property name="maximum">
           <number>8192</number>
          </property>
          <property name="value">
           <number>80</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>H:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_2">
          <property name="maximum">
           <number>8192</number>
          </property>
          <property name="value">
           <number>60</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="chbSizeAutomatic">
        <property name="text">
         <string>Automatic</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="chbSizeFixedWindow">
        <property name="text">
         <string>Fixed window</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Emulator</string>
     </property>
     <layout class="QFormLayout" name="formLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_5">
        <item>
         <property name="text">
          <string>Raw</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>VT1xx</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>X-Term</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Palette</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="grbFont">
     <property name="title">
      <string>Font</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="btnFontSelect">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QCheckBox" name="chbFontProportional">
          <property name="text">
           <string>Proportional</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="chbFontMonospace">
          <property name="text">
           <string>Monospace</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="chbFontScalable">
          <property name="text">
           <string>Scalable</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="chbFontNonScalable">
          <property name="text">
           <string>Non scalable</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="lblFontExample">
        <property name="text">
         <string>Example text, how it looks like</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="lblFontCurrentValue">
        <property name="text">
         <string>Current</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Encodding</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Receive</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_3">
        <item>
         <property name="text">
          <string>ASCII</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>UTF-8</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Transmit</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_4">
        <item>
         <property name="text">
          <string>ASCII</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>UTF-8</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Backspace/Delete</string>
     </property>
     <layout class="QFormLayout" name="formLayout_6">
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Backspace</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_6">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CTRL-H</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Backspace</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Delete</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_7">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CTRL-H</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Backspace</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Delete</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
